worker-workflow/src/main/resources/add-failures.js

```diff
@@ -24,10 +24,17 @@ function addFailures (document, failures, extractSourceCallback, action) {
         return JSON.parse(failureSubfieldsJson);
     }
 
+    function getWorkflowAction(document) {
+        var actionField = document.getRootDocument().getField("CAF_WORKFLOW_ACTION");
+        if (actionField && actionField.hasValues()) {
```

Comment:
`.getField()` will never return `null`.

```suggestion
        if (actionField.hasValues()) {
```